frontend/javascripts/libs/format_utils.ts

```diff
@@ -144,6 +144,36 @@ export function formatNumberToLength(lengthInNm: number): string {
   return formatNumberToUnit(lengthInNm, nmFactorToUnit);
 }
 
+const nmFactorToUnit2D = new Map([
+  [1e-6, "pm²"],
+  [1, "nm²"],
+  [1e6, "µm²"],
+  [1e12, "mm²"],
+  [1e18, "m²"],
+  [1e24, "km²"],
+]);
+/* TODO: or rather: 
+const nmFactorToUnit2D = new Map(
+  Array.from(nmFactorToUnit).map((entry) => [Math.pow(entry[0], 2), entry[1].concat("²")]),
+); 
+Although I think that the explicit maps are better to read and it's better to have them stored rather than computing them every time they are needed.
```

Comment:
TODO

Reply:
yes, I think it's fine to leave these hardcoded.